lib/new_relic/agent/instrumentation/rails_notifications/action_controller.rb

```diff
@@ -29,5 +30,14 @@
 
     NewRelic::Agent::Instrumentation::ActionControllerSubscriber \
       .subscribe(/^process_action.action_controller$/)
+
+    subs = %w[send_file
+      send_data
+      redirect_to
+      halted_callback
+      unpermitted_parameters]
+
+    NewRelic::Agent::Instrumentation::ActionControllerOtherSubscriber \
+      .subscribe(Regexp.new("^(#{subs.join('|')}).action_controller$"))
```

Comment:
The dynamic "OR"ing of the event names is neat to see! The literal period here should be escaped (`.` -> `\.`) so that the pattern will only allow a literal period and not any character.

Reply:
oh yeah, great point! I ended up having to double escape it though because it's starting as a string first, and then gets converted into a regex. 
53639679fc7b9da07442ed96b68449c2dbe983a7

Reply:
oh interesting - it's specifically the interpolation of the double quoted string that chomps up the single backslash if not prefixed by an additional backslash:

```ruby
irb(main):001:0> Regexp.new('a\.test')
=> /a\.test/
irb(main):002:0> Regexp.new("a\.test")
=> /a.test/
irb(main):003:0> Regexp.new("a\\.test")
=> /a\.test/
irb(main):004:0> /a\.test/
=> /a\.test/
```

Also on the topic of regexes for notification subscriptons, Kayla and I have been talking about standardizing on `\A` and `\z` instead of `^` and `$` and I'd also like to see us standardize on informing Ruby's regex engine that we don't actually want to capture the ORed list of words (so `(#{subs.join('|')})` would become `(?:#{subs.join('|')})`. But these 2 refactors can go in later post milestone.